<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_exam</class>
 <widget class="QDialog" name="Dialog_exam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>检验信息</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>531</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0">
     <widget class="QDoubleSpinBox" name="value_fg"/>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QDoubleSpinBox" name="value_pt"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_type">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Hei&quot;;</string>
      </property>
      <property name="text">
       <string>aptt</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_fg">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Hei&quot;;</string>
      </property>
      <property name="text">
       <string>fg</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_size">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Hei&quot;;</string>
      </property>
      <property name="text">
       <string>pt</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QDoubleSpinBox" name="value_aptt"/>
    </item>
    <item row="8" column="2" colspan="2">
     <widget class="QDoubleSpinBox" name="value_dd"/>
    </item>
    <item row="4" column="1">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEdit_patientID">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>病人ID</string>
      </property>
      <property name="placeholderText">
       <string>编号</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_dd">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Hei&quot;;</string>
      </property>
      <property name="text">
       <string>dd</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_examID">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>检验ID</string>
      </property>
      <property name="placeholderText">
       <string>姓名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_ID">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Hei&quot;;</string>
      </property>
      <property name="text">
       <string>病人ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_exam">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Hei&quot;;</string>
      </property>
      <property name="text">
       <string>检验ID</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <spacer name="horizontalSpacer_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1">
     <spacer name="horizontalSpacer_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_exam">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Hei&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="creat.qrc"/>
 </resources>
 <connections/>
</ui>
